pac4j-core/src/main/java/org/pac4j/core/client/IndirectClient.java

```diff
@@ -40,7 +40,7 @@ protected void internalInit(final WebContext context) {
     }
 
     @Override
-    public final HttpAction redirect(final WebContext context) throws HttpAction {
+    public HttpAction redirect(final WebContext context) throws HttpAction {
```

Comment:
The callback URL is checked by the `GoogleOAuthServiceImpl` which is a singleton I can't make it work by overriding the `Google2Client`.
In my case I have two callback URL: 
http://tenant1.com/callback?client_name=Google2Client
http://tenant2.com/callback?client_name=Google2Client

The authentication works with the first tenant but with the second I have this error:

> Caused by: com.github.scribejava.core.exceptions.OAuthException: Response body is incorrect. Can't extract a '"access_token"\s*:\s*"(\S*?)"' from this: '{
>  "error": "redirect_uri_mismatch",
>  "error_description": "Bad Request"
> }